documentation/design_docs/L1/msp432p401r/system_controller.md

```diff
@@ -12,14 +12,14 @@
     - [void UnlockClockSystemRegister() const](#void-unlockclocksystemregister-const)
     - [void LockClockSystemRegister() const](#void-lockclocksystemregister-const)
   - [Initialization](#initialization)
-    - [Returns\<void> Initialize() override](#returnsvoid-initialize-override)
-    - [Returns\<units::frequency::hertz_t> ConfigureDcoClock() const](#returnsunitsfrequencyhertzt-configuredcoclock-const)
-    - [Returns\<units::frequency::hertz_t> ConfigureReferenceClock() const](#returnsunitsfrequencyhertzt-configurereferenceclock-const)
-    - [Returns\<void> SetClockSource(Clock clock, Oscillator oscillator) const](#returnsvoid-setclocksourceclock-clock-oscillator-oscillator-const)
-    - [Returns\<void> SetClockDivider(Clock clock, ClockDivider divider) const](#returnsvoid-setclockdividerclock-clock-clockdivider-divider-const)
+    - [Returns&lt;void> Initialize() override](#returnsvoid-initialize-override)
```

Comment:
Using `&lt;` so that '&lt;' shows up in HTML.

Reply:
Awesome! I was wondering what we could do to make this show up.